src/processing.py

```diff
@@ -0,0 +1,18 @@
+def filter_by_state(list_dict: list[dict], state: str = "EXECUTED") -> list[dict]:
+    """
+    Функция принимает список словарей и возвращает новый список со словарями
+    у которых ключ state равен значению EXECUTED.
```

Comment:
Если в `state` передать другое значение - фильтрация пойдет по другому статусу
```suggestion
    у которых ключ state равен значению `state`.
```